google_guest_agent/network/manager/netplan_linux.go

```diff
@@ -144,7 +144,12 @@ func (n netplan) Name() string {
 
 // Configure gives the opportunity for the Service implementation to adjust its configuration
 // based on the Guest Agent configuration.
-func (n netplan) Configure(ctx context.Context, config *cfg.Sections) {
+func (n *netplan) Configure(ctx context.Context, config *cfg.Sections) {
+	osInfo := osinfoGet()
+	// Debian's default netplan configuration is priority 90.
+	if osInfo.OS == "debian" && osInfo.Version.Major == 12 {
+		n.priority = 91
```

Comment:
I believe this would make it so the default netplan config has higher priority due to it being lexicographically earlier.

Reply:
I take this back, it seems like later priorities take priority, as they are applied later and override previous keys if those keys exist.

https://netplan.io/faq#hierarchy-of-configuration-files